test/eth/SmartContractService.spec.js

```diff
@@ -15,65 +15,48 @@ test('getContractByName should have proper error checking', done => {
     'Cannot resolve network ID. Are you connected?'
   );
 
-  service
-    .manager()
-    .authenticate()
-    .then(() => {
-      expect(() =>
-        service.getContractByName(contracts.SAI_TOP, { version: 999 })
-      ).toThrow('Cannot find contract SAI_TOP, version 999');
```

Comment:
Keeping `expect ... toThrow` here is a bit more idiomatic Jest

Reply:
Nice, I didn't know you could do it this way with async functions but its much cleaner.

Reply:
`getContractByName` isn't async :)

Reply:
oops, totally whiffed on that one 🙄